compliance/api_test.py

```diff
@@ -38,13 +61,15 @@ def test_is_exports_verification_disabled(settings, key):
     indirect=["cybersource_mock_client_responses"],
 )
 def test_verify_user_with_exports(
-    user, cybersource_mock_client_responses, expected_result
+    user, cybersource_mock_client_responses, cybersource_private_key, expected_result
```

Comment:
`cybersource_private_key` isn't used and the test passes without it, so I think it can be removed.

Reply:
Fixed